craft_parts/executor/part_handler.py

```diff
@@ -743,7 +772,8 @@ def _migrate_overlay_files_to_prime(self) -> None:
         for part in reversed(parts_with_overlay):
             logger.debug("migrate part %r layer to prime", part.name)
             visible_files, visible_dirs = overlays.visible_in_layer(
-                part.part_layer_dir, part.prime_dir
+                part.part_layer_dir,
```

Comment:
do you know which rule is splitting lines like this one?

Reply:
that come from the `ruff format', which I think has no rule code

Reply:
OK, after some tests, this kind of format was done by `black`, not `ruff format`.